src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -664,7 +664,9 @@ private IOperation CreateBoundObjectInitializerMemberOperation(BoundObjectInitia
                     }
                     else
                     {
-                        var accessor = property.GetOwnOrInheritedSetMethod();
+                        // In collection initializers, the property is not actually set. Instead, it is retrieved for a series of Add methods, so we need
```

Comment:
>// In collection initializers, the property is not actually set. Instead, it is retrieved for a series of Add methods, so we need [](start = 24, length = 129)

I think this should also apply to the case when a property is initialized with a nested initializer. Is this scenario handled properly?